.eslintrc

```diff
@@ -13,6 +13,7 @@
 		"jQuery": true,
 		"wp": false
 	},
+	"extends": 'wordpress',
```

Comment:
JSON only allows double-quotes for strings. This will be a parse-error in a pure JSON parser.